src/base/installDevTools.m

```diff
@@ -0,0 +1,105 @@
+function [] = installDevTools()
+% checks if the MATLAB.devTools are already present and installs the
+% MATLAB.devTools next to the cloned folder cobratoolbox (if not present yet)
+%
+% USAGE:
+%    installDevTools()
+%
+% NOTE:
+%    In order to install the devTools, please make sure that the SSH key is
+%    configured properly as explained `here`_.
+%
+% .. _here: https://opencobra.github.io/cobratoolbox/docs/solvers.html
+%
+% .. Author: Laurent Heirendt, June 2017
+
+    global ENV_VARS
+    global CBTDIR
+
+    % make sure that The COBRA Toolbox is properly installed;
+    % git and curl are tested here
+    if isempty(ENV_VARS)
+        ENV_VARS.printLevel = false;
+        initCobraToolbox;
+        ENV_VARS.printLevel = true;
+    end
+
+    currentDir = pwd;
+
+    % change to the local directory
+    localDir = [CBTDIR filesep '..'];
+    cd(localDir);
+
+    installFlag = false;
+
+    % check if the devTools already exist
+    if exist([CBTDIR filesep '..' filesep 'MATLAB.devTools'], 'dir') == 7
+
+        reply = input([' > There is already a copy of the MATLAB.devTools installed in the default location.\n   Do you want to enter another location to install the MATLAB.devTools? Y/N [Y]: '], 's');
+        localDir = '';
+
+        if isempty(reply) || strcmpi(reply, 'y') || strcmpi(reply, 'yes')
+        % enter another location for the MATLAB.devTools
+
+            dirReply = '';
+            while isempty(dirReply)
+                dirReply = input(['\n -> Please define the installation path of the MATLAB.devTools\n    Enter the path: '], 's');
```

Comment:
It's just a matter of preference :) 
As a programmer: I completely agree, from a users point of view typing in a directory is a pain. ^^